src/components/Tweet.js

```diff
@@ -1,23 +1,18 @@
+import ProfileImage from "./ProfileImage";
+import User from "./User";
+import Timestamp from "./Timestamp"
+
 function Tweet(props) {
 
   const {tweet} = props
-  console.log(tweet)
   return (
     <div className="tweet">
-      <img
-        src={tweet.user.image}
-        className="profile"
-        alt="profile"
-      />
+      <ProfileImage image={tweet.user.image}/>
 
       <div className="body">
         <div className="top">
```

Comment:
This div's should be inside the respective components. The point of having a lot of components is to have a cleaner file:
```
function Tweet(props) {
  return (
    <div className="tweet">
      <ProfileImage image={props.tweet.user.image} />
      <User userData={props.tweet.user} />
      <Timestamp time={props.tweet.timestamp} />
      <Message message={props.tweet.message} />
      <Actions />
      <i className="fas fa-ellipsis-h"></i>
    </div>
  );
}
```